src/const/meta.ts

```diff
@@ -1,9 +1,11 @@
-const API_BASE_URL = "https://www.mevblocker.com"
+const API_BASE_URL = "https://www.mevblocker.com";
 
 export const CONFIG = {
-  title: 'MEV Blcoker',
-  description: 'CoW Protocol finds the lowest prices from all decentralized exchanges and DEX aggregators & saves you more with p2p trading and protection from MEV ',
-  descriptionShort: 'The smartest way to trade',
+  title: "MEV Blcoker",
+  description:
+    "CoW Protocol finds the lowest prices from all decentralized exchanges and DEX aggregators & saves you more with p2p trading and protection from MEV ",
```

Comment:
Since this is not exactly CoW Protocol, shouldn't we use a different sentence?
Same comment for the short description

(I know it was not added by you in this PR, but raising the point to get the discussion started)